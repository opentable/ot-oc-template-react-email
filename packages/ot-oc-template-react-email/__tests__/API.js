const template = require("../index.js");

test("should expose the correct methods", () => {
  expect(template).toMatchSnapshot();
});
