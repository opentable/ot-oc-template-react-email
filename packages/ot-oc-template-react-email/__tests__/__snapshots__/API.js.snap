// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should expose the correct methods 1`] = `
Object {
  "getCompiledTemplate": [Function],
  "getInfo": [Function],
  "render": [Function],
}
`;
